File: app/src/main/java/com/lucas/hometask/model/Usuario.java
package com.lucas.hometask.model;

import java.io.Serializable;

public class Usuario implements Serializable {
    private String id;
    private String nome;
    private String email;
    private String idCasa;
    private String imagem;

    public Usuario() {
    }

    public Usuario(String nome, String email) {
        this.nome = nome;
        this.email = email;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getIdCasa() {
        return idCasa;
    }

    public void setIdCasa(String idCasa) {
        this.idCasa = idCasa;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getImagem() {
        return imagem;
    }

    public void setImagem(String imagem) {
        this.imagem = imagem;
    }
}

File: app/src/main/java/com/lucas/hometask/model/Usuario.kt
package com.lucas.hometask.model

import java.io.Serializable

class Usuario : Serializable {
    var id: String? = null
    var nome: String? = null
    var email: String? = null
    var idCasa: String? = null
    var imagem: String? = null

    constructor()

    constructor(nome: String?, email: String?) {
        this.nome = nome
        this.email = email
    }
}
